### DISCUSSION TITLE
GitHub Educator approval process is way too burdensome

### DISCUSSION THREAD

**Discussion by timwonderer** [community/community#84692] [GitHub Education] [answered]
### Hi everyone,

I am trying to apply for the teacher discount for GitHub Education and the process is beyond frustrating. I am not sure how else I can prove my status when:
1. Our school district (LAUSD, second largest in the US) does not give us .edu email but instead use .net
2. Our school is not on the dropdown menu when entering the name
3. There is no centralized employee ID card for our district
4. GitHub doesn't take paystub, LAUSD employment verification letter, etc.

I am not sure what else to do? Any pointers?

**Comment by AbdulConsole** [comment, marked as answer]
I understand your frustration.
It's best to directly reach out to GitHub Education support. And explain your specific circumstances and inquire about alternative documentation options or steps you can take to verify your teacher status.

They might be able to provide guidance or offer solutions tailored to your situation.

Follow their support line here https://support.github.com/request/education?tags=dotcom-direct

Please mark this as an answer.
